src/components/Toaster/index.vue

```diff
@@ -73,3 +74,10 @@ export default {
   border-color: transparent!important;
 }
 </style>
+
+<style lang="scss">
+// BS only provides overflow-[auto, hidden]
+.toast.overflow-unset {
```

Comment:
put it inside scoped scss and use ```/deep/```

Reply:
That was my initial plan but there is a slight problem. Bootstrap doesn't show it's toasts in this component, but rather portals them into its own toaster component that is located somewhere else.

( with this: https://www.npmjs.com/package/portal-vue )

I'll take another look (later) if I can figure out something scoped.

Reply:
if you'll remove ```.border-transparent``` class from scoped scss (and I'm pretty sure it's necessary as you can use ```.border-0```) then it's ok to leave this one as it is.